CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Development/contribution Process
+
+Our development process is based on [GitHub Flow](http://scottchacon.com/2011/08/31/github-flow.html) (highly recommended read if you're not familiar with it).  Merging to the master branch requires approval by **@pandell/developers**; other projects may have different approvers.
+
+Typical steps:
+
+0. Create a private branch. Commit all of your changes to this branch.  Commit messages should follow [these guidelines](https://github.com/pandell/SamplePliWeb/wiki/Writing-good-commit-messages).  Each separate bug or feature you implement should be in a separate branch.  While you are working on a change, the corresponding issue in GitHub should be moved to the `in progress` label.
```

Comment:
This should be re-worded for public contributions (where reporistory is forked, because 3rd party developers don't have permissions to create private branches).